test/parallel/test-vm-strict-mode.js

```diff
@@ -0,0 +1,23 @@
+// https://github.com/nodejs/node/issues/5344
+'use strict';
+require('../common');
+const assert = require('assert');
+const vm = require('vm');
+const ctx = vm.createContext();
+
+vm.runInContext('w = 1;', ctx);
```

Comment:
minor nit... would you mind adding a short comment here about what precisely this test is verifying? ... just to make it easier for other devs who are looking at this test in the future.